test/unit/helpers/mocks/Util.js

```diff
@@ -64,6 +65,8 @@ fn.mockDuo = function () {
 };
 
 fn.mockAjax = function (responses) {
+  globalFetch = window.fetch;
+  window.fetch = null;
```

Comment:
Why was this change made?

Reply:
since we mock ajax requests and dont want to use fetch we need to force using ajax and not fetch.